src/video/windows/SDL_windowsevents.c

```diff
@@ -785,37 +786,29 @@ void WIN_PollRawInput(SDL_VideoDevice *_this)
         }
     }
 
-    now = SDL_GetTicksNS();
     if (total > 0) {
-        Uint64 mouse_timestamp, mouse_increment;
-        Uint64 delta = (now - data->last_rawinput_poll);
-        UINT total_mouse = 0;
+        bool idleless = (idle_end - idle_start) <= SDL_US_TO_NS(125);
```

Comment:
What does idleless mean? Also, maybe you can move the logic for this bool into the calling code, and change the 125 into a constant with a comment about why that number is a good one?